src/test/java/seedu/address/model/concert/ConcertContactTest.java

```diff
@@ -3,29 +3,77 @@
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertFalse;
 import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.testutil.TypicalConcertContacts.ALICE_COACHELLA;
+import static seedu.address.testutil.TypicalConcertContacts.ALICE_GLASTONBURY;
+import static seedu.address.testutil.TypicalConcertContacts.BOB_COACHELLA;
 import static seedu.address.testutil.TypicalConcerts.COACHELLA;
+import static seedu.address.testutil.TypicalConcerts.GLASTONBURY;
 import static seedu.address.testutil.TypicalPersons.ALICE;
 import static seedu.address.testutil.TypicalPersons.BOB;
 
 import org.junit.jupiter.api.Test;
 
+import seedu.address.model.person.Person;
+
 public class ConcertContactTest {
     @Test
     void createConcertContact() {
-        //checks if ConcertContact object can be created
+        // checks if ConcertContact object can be created
         ConcertContact contactTest1 = new ConcertContact(ALICE, COACHELLA);
 
         assertEquals(COACHELLA, contactTest1.getConcert());
         assertEquals(ALICE, contactTest1.getPerson());
     }
 
     @Test
-    void equalsTest() {
+    public void isSameConcertContact() {
+        // same concert returns true
+        assertTrue(ALICE_COACHELLA.isSameConcertContact(ALICE_COACHELLA));
+
+        // different person returns false
+        assertFalse(ALICE_COACHELLA.isSameConcertContact(BOB_COACHELLA));
+
+        // different concert returns false
+        assertFalse(ALICE_COACHELLA.isSameConcertContact(ALICE_GLASTONBURY));
+    }
+
+    @Test
+    public void isAssociated_person() {
```

Comment:
add test for isSamePerson but not equals. E.g. same person name but different address. Important as the isAssociated should match the exact Person, not just the name field